.travis.yml

```diff
@@ -11,12 +11,9 @@ before_cache:
   - find $HOME/.sbt -name "*.lock" -delete
 matrix:
   include:
-    - jdk: openjdk8
-      scala: 2.11.12
-      env: HADOOP_VERSION="2.7.7" SPARK_VERSION="2.4.5" AWS_JAVA_SDK_VERSION="1.7.4"
```

Comment:
should we keep testing for this ?

Reply:
Yes, this is also my concern, spark 2.x needs this to be passed. What do you think?

Reply:
Let's move on entirely on spark 3 in this PR.